client/js/components/map/map/DpOlMap.vue

```diff
@@ -440,6 +440,8 @@ export default {
     this.centerX = (this.maxExtent[0] + this.maxExtent[2]) / 2
     this.centerY = (this.maxExtent[1] + this.maxExtent[3]) / 2
 
+    this.options.defaultAttribution =  mapOptions.copyright
```

Comment:
In some places, `options.defaultAttribution` is set via prop from the parent component. While we need to preserve the ability to change that prop client side dynamically, the approach to set a default via api is a handy improvement.

I'd suggest to use `this._options` instead of `this.options` (just for consistency, because this._options already [contains the local defaults](https://github.com/demos-europe/demosplan-core/blob/9268027b23f06bf302c1195276ea321e8f757d46/client/js/components/map/map/DpOlMap.vue#L416). [This line](https://github.com/demos-europe/demosplan-core/blob/9268027b23f06bf302c1195276ea321e8f757d46/client/js/components/map/map/DpOlMap.vue#L98) has to be adapted accordingly.

Untested but the suggestion should lead to the prop value being respected while using the default of mapOptions.copyright:

```suggestion
    this._options.defaultAttribution = this.options?.defaultAttribution || mapOptions.copyright
```

Reply:
hey @spiess-demos thank you for the feedback. I will do it. I realized what you said:

1. DpMapMpdel.vue passes "mapOptionsRoute" parameter to DpOlMap.  "mapOptionsRout contains the the api url to call the BE
2. Then the DpOlMap.vue calls the BE using that api url (mapOptionsRoute) and gets several info to adjust the map (including the copyright of the map)
3.  Then what I did in DpOlMap.vue is to set the copyright into "options.defaultAttribution", because DpOlMap.vue passes "options" as parameter to DpOlMapLayer.vue, and ultimaltey DpOlMapLayer.vue uses "options.defaultAttribution" value to set the copyright text on the right corner of the map.

Reply:
@spiess-demos huhu Hans, I adjusted the changes and it does not work, the copyright coming from BE is not taken